TwitchClient/Channel.swift

```diff
@@ -1,27 +0,0 @@
-//
-//  Channel.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/8/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Channel: NSObject {
-    
-    var status : String?
-    var language : String?
-    var displayName : String?
-    var logo : String?
```

Comment:
`logoURL : URL` maybe?

Topic to read in general: Naming in programming in general and also any Swift guidelines doc.